configs/config-tls-generic.h

```diff
@@ -402,10 +402,14 @@
 #define MBEDTLS_PK_PARSE_C
 #endif
 
-#if defined(MBEDTLS_PK_PARSE_C)
+#if defined(MBEDTLS_PK_PARSE_C) || defined(CONFIG_MBEDTLS_PK_C)
```

Comment:
I wonder if we need both, `CONFIG_MBEDTLS_PK_C` and `CONFIG_MBEDTLS_PK_WRITE_C`. Sure, it gives more flexibility, but also creates a risk of selecting an invalid configuration. I'd rather consider enabling PK automatically if PK_WRITE is enabled:
```suggestion
#if defined(MBEDTLS_PK_PARSE_C) || defined(MBEDTLS_PK_WRITE_C)
```

Reply:
Makes sense.